### DISCUSSION TITLE
Invariant Violation: requireNativeComponent: "RNSScreenStackHeaderConfig" was not found in the UIManager.

### DISCUSSION THREAD

**Discussion by ToniNikolaev23** [software-mansion/react-native-screens#1613] [Q&A] [answered]
Hello today i run my react native project and i have this error : Invariant Violation: requireNativeComponent: "RNSScreenStackHeaderConfig" was not found in the UIManager.
This is my package.json
 "react-native-gesture-handler": "^2.4.1",
    "react-native-reanimated": "2.8.0",
    "react-native-render-html": "^6.3.4",
    "react-native-safe-area-context": "^4.2.4",
    "react-native-screens": "^3.13.1",

React-native-screens in yarn.lock is installed last version -> 3.18.1 and i get this error.
After change "react-native-screens": "3.18.0" and build -> app works

Yarn install with "react-native-screens": "3.18.1" -> app not work
Yarn install with "react-native-screens": "3.18.0" -> app work

**Comment by tboba** [comment, marked as answer]
Hi @ToniNikolaev23, thanks for submitting this problem! Do you know if this problem still occurs on the newest react-native-screens (currently 3.26.0)? If so, let's move to the issue, related to that problem: #1614.

Thanks!
